src/commands/commerce/extension/points/list.ts

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { SfdxCommand } from '@salesforce/command';
+import { Messages, Org } from '@salesforce/core';
+import { forceDataSoql } from '../../../../lib/utils/sfdx/forceDataSoql';
+
+Messages.importMessagesDirectory(__dirname);
+
+const TOPIC = 'extension';
+const CMD = `commerce:${TOPIC}:getEPN`;
+const msgs = Messages.loadMessages('@salesforce/commerce', 'store');
+
+export class getEPN extends SfdxCommand {
+    public static readonly requiresUsername = true;
+    public static example = [`sfdx ${CMD}`];
+
+    public static description = msgs.getMessage('extension.getEPN.cmdDescription');
+    public org: Org;
+
+    // eslint-disable-next-line @typescript-eslint/require-await
+    public async run(): Promise<void> {
+        this.ux.log(`Getting all ExtensionPointName values using username: ${this.org.getUsername()} \n..........`);
+        const domainInfo = forceDataSoql(
+            "SELECT Value, Label, IsDefaultValue, IsActive FROM PicklistValueInfo WHERE EntityParticle.DurableId = 'RegisteredExternalService.ExtensionPointName'"
+        );
+        // iterates through picklist to show availabe EPN vals
+        this.ux.log('PICKLIST Value Table:', '\n');
+
+        for (const element of domainInfo.result.records) {
```

Comment:
what happens when the table is empty? the command should return an empty result instead of throwing an error.  Please add unit tests for the list command

Reply:
It returns an empty result. Blank table